setupext.py

```diff
@@ -1386,7 +1386,7 @@ def check(self):
                 import functools32
             except ImportError:
                 return (
-                    "functools32 was not found. It is required for for"
+                    "functools32 was not found. It is required for"
```

Comment:
I think you didn't mean to change this one and were looking for the double for in the subprocess32 package, but since you're here, it should say Python.